### ISSUE TITLE
Touch Not enabled?

### ISSUE THREAD

**Issue by dtapuska** [GoogleChrome/wptdashboard#88]
The failures for Chrome:
http://wpt.fyi/touch-events/create-touch-touchlist.html
and
http://wpt.fyi/touch-events/touch-globaleventhandler-interface.html

indicate the bot is being launched without touch-enabled.

@RByers @MustaqAhmed

**Comment by RByers** [comment]
Yeah, that's the default configuration for Chrome, though should probably [be tweaked](https://bugs.chromium.org/p/chromium/issues/detail?id=392584).

I think the fundamental issue here is that we don't run on Android (or in a mode where Chrome desktop is attempting to emulate Android).  We do have some other APIs which are exposed only on mobile (eg. PaymentRequest), perhaps we should consider trying to get Chrome Android results?

As a short-term hack for this specific issue we could run with --touch-events=enabled, but that doesn't really address the root issue.

**Comment by RByers** [comment]
Hah hah - Twitter just notified me that [PaymentRequest is now on desktop](https://twitter.com/agektmr/status/905289098365210625).
